mpmath/ctx_fp.py

```diff
@@ -230,10 +231,6 @@ def harmonic(ctx, x):
     def to_fixed(ctx, x, prec):
         return int(math.ldexp(x, prec))
 
-    def rand(ctx):
```

Comment:
I don't know why this was replaced with the code below: new `ctx.convert` call is just a source of an additional slowdown.

I think this should be reverted.

Reply:
You're right. I reverted this.